### PR TITLE
GVL For TCF2 Allows Empty Vendor List

### PR DESCRIPTION
**Pull request by SyntaxNode**
Resolves https://github.com/prebid/prebid-server/issues/1689

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZbAQM=
+github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
+github.com/stretchr/objx v0.1.0/go.mod h1:HFkY916IF+rwdDfMAkV7OtwuqBVzrE8GR6GFx+wExME=
+github.com/stretchr/testify v1.7.0 h1:nwc3DEeHmmLAfoZucVR881uASk0Mfjw8xYJ99tb5CcY=
+github.com/stretchr/testify v1.7.0/go.mod h1:6Fq8oRcR53rry900zMqJjRRixrwX3KX962/h/Wwjteg=
+gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
+gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c h1:dUUwHk2QECo/6vqA44rthZ8ie2QXMNeKRTHCNY2nXvo=
+gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c/go.mod h1:K4uyk7z7BCEPqu6E+C64Yfv1cQ7kz7rIZviUmN+EgEM=
```
Comment (SyntaxNode) [inline go.sum:12]: These are all indirect dependencies from Testify.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 language: go
 go:
-    - "1.14"
-    - "1.12"
```
Comment (SyntaxNode) [inline .travis.yml:4]: Testify no longer supports Go 1.12. Think it's time to move on, or is it important to keep Go 1.12 compatibility?

Comment (hhhjort) [reply .travis.yml:4]: Move on I think

**Review by hhhjort — APPROVED** [review]
LGTM

**Review by bsardo — APPROVED** [review]

**Pull request merged**